src/Kernel.php

```diff
@@ -146,20 +132,14 @@ function loadState(SnapshotStore $snapshotStore, Message $message, AggregateDefi
     return $aggregate->aggregateRoot();
 }
 
-const loadEvents = 'Prooph\Micro\Kernel\loadEvents';
+const loadEvents = '\\Prooph\\Micro\\Kernel\\loadEvents';
```

Comment:
So cool that you discovered the `const` definition pattern 😄!
With single quotes, you don't need to worry about double backslashes.